ibis/__init__.py

```diff
@@ -40,6 +40,11 @@
     pass
 
 try:
+    import ibis.bigquery.api as bigquery
+except ImportError:  # pip install ibis-framework[bigquery]
+    pass
```

Comment:
need to add the pre-reqs to ci/requirements-2.7/3.6 (leave 3.4/4.5 so that they skip). need to mod appveyor.yml (see #1167 for what you need to add).

Reply:
addressed in [ TST: add "git+https://github.com/googledatalab/..." as requirement ](https://github.com/ibis-project/ibis/pull/1170/commits/2965c451f06a80c9ab6c6c97a9db82fa4dfb13ee)

I don't think there's a pip package for pydatalab yet, so only have the github.com url